test/integration/utility.cc

```diff
@@ -114,15 +114,22 @@ IntegrationUtil::makeSingleRequest(uint32_t port, const std::string& method, con
 
 RawConnectionDriver::RawConnectionDriver(uint32_t port, Buffer::Instance& initial_data,
                                          ReadCallback data_callback,
-                                         Network::Address::IpVersion version) {
+                                         Network::Address::IpVersion version,
+                                         Event::Dispatcher& dispatcher,
```

Comment:
Would it be better to have a method on BaseIntegrationTest that created a raw connection driver and implicitly passed the test dispatcher? Might be less change?

Reply:
Great idea! Added that helper and use it everywhere